brew/src/main/java/nl/tudelft/serg/evosql/brew/generator/junit/JUnitGenerator.java

```diff
@@ -342,11 +347,11 @@ private MethodSpec generatePathTest(Path path, VendorOptions vendorOptions) {
 
         // Act
         pTestBuilder.addComment("Act: run a selection query on the database");
-        pTestBuilder.addStatement("$T result = runSQL($L, false)", boolean.class, NAME_PRODUCTION_QUERY);
+        pTestBuilder.addStatement("$T result = runSQL($L, false)", int.class, NAME_PRODUCTION_QUERY);
 
         // Assert
-        pTestBuilder.addComment("Assert: verify that at least one resulting row was returned");
-        pTestBuilder.addStatement("$T.assertEquals($L, result)", assertionClass, path.isSuccess());
+        pTestBuilder.addComment("Assert: verify that the expected amount of rows is returned");
```

Comment:
(super nitpick) Grammatically speaking, "amount" should be "number" since it's a countable quantity. It's not a blocking issue in my opinion.